src/Native/Unix/System.Native/pal_console.c

```diff
@@ -257,10 +289,9 @@ void SystemNative_GetControlCharacters(
 
 int32_t SystemNative_StdinReady()
 {
-    SystemNative_InitializeConsoleBeforeRead(1, 0);
+    // TODO ??: https://github.com/dotnet/corefx/pull/6619
```

Comment:
I'm not sure if I need to do something here other than poll. I don't know what the InitializeConsoleBeforeRead that was here before was meant to do.